libs/embed/Dockerfile

```diff
@@ -14,14 +14,18 @@ COPY packages/client ./packages/client
 COPY libs/embed ./libs/embed
 COPY packages/notification-center ./packages/notification-center
 
+#COPY ["tsconfig.json","tsconfig.base.json","nx.json","pnpm-workspace.yaml","pnpm-lock.yaml", "./"]
+
 COPY tsconfig.json .
 COPY tsconfig.base.json .
 
 COPY nx.json .
 COPY pnpm-workspace.yaml .
 COPY pnpm-lock.yaml .
 
-RUN pnpm install
+RUN pnpm install --reporter=silent
 RUN pnpm build
 
+#RUN chown -R 1000:root .
+
 CMD [ "pnpm", "start:docker:embed" ]
```

Comment:
This is correct, it happens because we allow modifying the the ENV variables of the API_URL in runtime of the dockerfile, need to find a smarter we to handle this later :|